automate_server.tf

```diff
@@ -12,7 +12,7 @@ variable "contact_tag" {
 }
 
 provider "aws" {
-    region = "us-west-2"
+  region     = "${var.region}"
```

Comment:
See above - I do not agree with making this a variable currently... there's no current benefit and other changes will need to be made before it can truly be supported.